pkg/util/tracing/tracingservicepb/tracing_service.proto

```diff
@@ -20,7 +20,10 @@ message SpanRecordingRequest {
 }
 
 message SpanRecordingResponse {
-  repeated tracing.tracingpb.RecordedSpan span_recordings = 1 [(gogoproto.nullable) = false];
+  message Recording {
+    repeated tracing.tracingpb.RecordedSpan recorded_spans = 1 [(gogoproto.nullable) = false];
+  }
+  repeated Recording recordings = 1 [(gogoproto.nullable) = false];
 }
 
 service Tracing {
```

Comment:
Worth adding a comment to GetSpanRecordings (and SpanRecordingRequest) to explain what they do, and what the "structure" of the return type is. Related: should it be called GetSpanRecordings{Request,Response} instead?

Reply:
yup, done.